### PR TITLE
Allow checking permissions of Entities

### PR DESCRIPTION
**Pull request by unilock**
As opposed to a `ServerPlayerEntity` derived from `ServerCommandSource`.
(there may be a more efficient way to do this)

### CODE REVIEW

File: src/main/java/dev/codedsakura/blossom/lib/permissions/Permissions.java [hunk 1]
```diff
@@ -62,4 +62,14 @@ public static boolean check(@NotNull ServerCommandSource source, @NotNull String
...
+
+    /**
+     * @see me.lucko.fabric.api.permissions.v0.Permissions#check(Entity, String, boolean)
+     */
+    public static boolean check(@NotNull Entity entity, @NotNull String permission, boolean fallback) {
+        if (isFPAPILoaded()) {
+            return PermissionsExecutor.check(entity, permission, fallback);
+        }
+        return fallback;
+    }
```
Comment (CodedSakura) [inline src/main/java/dev/codedsakura/blossom/lib/permissions/Permissions.java:74]: i think a `check(Entity, String, int)` would be useful aswell, same in the `PermissionsExecutor` file

**Review by CodedSakura — CHANGES_REQUESTED** [review]
please also bump the mod version to 2.5.0

File: src/main/java/dev/codedsakura/blossom/lib/permissions/Permissions.java (hunk 1 above)
Comment (unilock) [reply src/main/java/dev/codedsakura/blossom/lib/permissions/Permissions.java:74]: not sure why i didn't do that already 🤔

**Review by CodedSakura — APPROVED** [review]
:+1:

**Pull request merged**